sw/otbn/code-snippets/p256.s

```diff
@@ -659,39 +659,39 @@ ModInvVar:
 impvt_Loop:
   bn.or w4, w4, w4
   csrrs x2, 1984, x0
-  andi x2, x2, 2
+  andi x2, x2, 4
```

Comment:
I wonder whether we should expose some "enum" at the assembly level to avoid needing the magic numbers. Something like "M_FLAG" or similar. What do you think?